client/src/components/QA/AnswerModal.jsx

```diff
@@ -95,49 +163,56 @@ class AnswerModal extends React.Component {
                 value={this.state.answer}
                 error={!!this.state.answerError}
                 helperText={this.state.answerError}
-                onChange={e => this.setState({answer: e.target.value})}
+                onChange={this.handleInput}
                 required>
               </TextField>
             </Stack>
             <br></br>
             <Stack spacing={1}>
               <TextField
+                id='Email'
                 label='Email'
                 placeholder="example@atelier.com"
                 fullWidth
                 inputProps={{maxLength: 60}}
                 value={this.state.email}
                 error={!!this.state.emailError}
                 helperText={this.state.emailError}
-                onChange={e => this.setState({email: e.target.value})}
+                onChange={this.handleInput}
                 required>
               </TextField>
               <p>For authentication reasons, you will not be emailed</p>
             </Stack>
             <br></br>
             <Stack spacing={1}>
               <TextField
+                id='Nickname'
                 label='Nickname'
                 placeholder="Howard878"
                 inputProps={{maxLength: 60}}
                 value={this.state.nickname}
                 error={!!this.state.nicknameError}
                 helperText={this.state.nicknameError}
-                onChange={e => this.setState({nickname: e.target.value})}
+                onChange={this.handleInput}
                 required>
               </TextField>
               <p>For privacy reasons, do not use your full name or email address</p>
             </Stack>
             <br></br>
+            <Stack direction='row' spacing={1}>
+              {this.state.images.map((image, index) => <img id='thumbnail' src={image}></img>)}
```

Comment:
An `id` should only appear once per page! 

This should be changed to a class name. 

**Required change. Different browsers will handle this differently and unpredictably!**

Reply:
Not using 'id' anymore.